packages/itwin/tree-widget/src/components/SelectableTree.tsx

```diff
@@ -61,13 +62,13 @@ function SelectableTreeContent(props: SelectableTreeProps & { imodel: IModelConn
 
   return (
     <div className="tree-widget-selectable-tree">
-      <SelectableContent {...getSelectableContentProps(trees)} />
+      <TreeSelector {...getTreeSelectorProps(trees, props.density)} />
```

Comment:
No need to pass `density` to `getTreeSelectorProps`. You can do `<TreeSelector {...getTreeSelectorProps(trees)} density={props.density} />`